tests/test_suites/ShortSystemTests/test_reads_during_small_ec_files_write_alternated.sh

```diff
@@ -0,0 +1,39 @@
+timeout_set 120 seconds
```

Comment:
How much time is consuming this test? 120 seconds is a long time.

Reply:
Done